<launch>
  <node pkg="stargazer_ros_tool" type="landmark_calibrator" name="landmark_calibrator" output="screen" required="true">
    <param name="stargazer_cfg_file_in" value="$(find stargazer_ros_tool)/res/stargazer.yaml"/>
    <param name="stargazer_cfg_file_out" value="$(find stargazer_ros_tool)/res/stargazer_optimized.yaml"/>
    <param name="bag_file" value="$(find stargazer_ros_tool)/../Stargazer.bag"/>
    <param name="landmark_topic" value="/landmarks_seen"/>
    <param name="pose_topic" value="/stargazer/camera_pose"/>
  </node>
</launch>
